dpnp/tests/test_umath.py

```diff
@@ -108,64 +109,6 @@ def test_umaths(test_cases):
     assert_allclose(result, expected, rtol=1e-6)
 
 
```

Comment:
Do we need to remove `@pytest.mark.usefixtures("allow_fall_back_on_numpy")` prior `test_umaths`?